ietf/static/img/linkedin-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="rgb(248, 249, 250)" class="bi bi-linkedin" viewBox="0 0 16 16">
```

Comment:
For single-colour icons it's usually preferable to set the fill(s) as `currentColor`, then use CSS to set/change the colour. That way you can change it easily on hover or other contexts.

Reply:
leaving it as is per our discussion